addons/web/static/lib/pdfjs/web/viewer.css

```diff
@@ -1096,6 +1096,73 @@
   --secondaryToolbarButton-spreadOdd-icon: url(images/secondaryToolbarButton-spreadOdd.svg);
   --secondaryToolbarButton-spreadEven-icon: url(images/secondaryToolbarButton-spreadEven.svg);
   --secondaryToolbarButton-documentProperties-icon: url(images/secondaryToolbarButton-documentProperties.svg);
+
+
+  --gray-100-rgb: 249, 250, 251;
```

Comment:
We can't touch the library itself. 
What you can do is adding another file to be loaded immediately after and place your overrides there ;)